tools/devops/automation/templates/governance/apiscan.yml

```diff
@@ -0,0 +1,28 @@
+parameters:
+
+- name: isPR
+  type: boolean
+
+- name: repositoryAlias
+  type: string
+  default: self
+
+- name: commit
+  type: string
+  default: HEAD
+
+- name: uploadPrefix
+  type: string
+  default: '$(MaciosUploadPrefix)'
+
+steps:
+
+- template: ../common/checkout.yml
+  parameters:
+    isPR: ${{ parameters.isPR }}
+    repositoryAlias: ${{ parameters.repositoryAlias }}
+    commit: ${{ parameters.commit }}
+
+- pwsh: |
+    Write-Output "Performing APISCan for $(PLATFORM) using tenant $(TENANT) client id $(CLIENT_ID) and secret $(SECRET)."
```

Comment:
Should the secret be displayed like this?

Reply:
Devops is super smart (for once), and knows not to display them:

<img width="1359" alt="Screenshot 2024-02-02 at 15 29 11" src="https://github.com/xamarin/xamarin-macios/assets/2190086/4853fd7f-46ef-4971-871c-a7557278dbb8">